linodecli/plugins/obj/__init__.py

```diff
@@ -553,15 +539,12 @@ def _configure_plugin(client: CLI):
     """
     Configures a default cluster value.
     """
-    clusters = get_available_cluster(client)
 
-    cluster = _default_thing_input(  # pylint: disable=protected-access
+    cluster = _default_text_input(  # pylint: disable=protected-access
         "Configure a default Cluster for operations.",
```

Comment:
nitpick: Small wording change to stay consistent with the [main CLI prompts](https://github.com/linode/linode-cli/blob/dev/linodecli/configuration/config.py#L409)
```suggestion
        "Default cluster for operations (e.g. `us-mia-1`)",
```